src/authorization/AuthorizationsDefinition.ts

```diff
@@ -82,7 +82,7 @@ export const AUTHORIZATION_DEFINITION: AuthorizationDefinition = {
           'fastChargePowerMax', 'drivetrainPowerHP'
         ]
       },
-      { resource: Entity.CAR_CATALOG, action: Action.SYNCHRONIZE },
+      { resource: Entity.CAR_CATALOG, action: [Action.SYNCHRONIZE, Action.DEBUG] },
```

Comment:
Debug??? :-)

Reply:
Yeah the superadmin has a tab called "advanced" where we display a json of the current car catalog item:
![image](https://user-images.githubusercontent.com/1226644/151848107-b73efb6f-3e5e-4299-bf69-f82f3516bdad.png)
Maybe we should remove this tab entirely ?

Reply:
LOL, I forgot this one!!!
Yes remove the tab :-) and the projection to not retrieve it from the DB.
Thanks!